Week1/homework/app.js

```diff
@@ -3,9 +3,173 @@
+      title: 'ince_memed',
+      language: 'turkish',
+      author: 'yasar_kemal',
+      desc:
+        'Memed, My Hawk (Turkish: İnce Memed, meaning "Memed, the Slim") is a 1955 novel by Yaşar Kemal. It was Kemal\'s debut novel and is the first novel in his İnce Memed tetralogy. The novel won the Varlik prize for that year (Turkey\'s highest literary prize) and earned Kemal a national reputation. In 1961, the book was translated into English by Edouard Roditi, thus gaining Kemal his first exposure to English-speaking readers. ',
+      imgPath: 'http://images.ykykultur.com.tr/upload/image/delta_yasar_kemal_ince_memed-5875.jpg'
+    },
+    {
+      id: 8,
+      title: 'kurk_mantolu_madonna',
+      language: 'turkish',
+      author: 'sabahattin_ali',
+      desc:
+        "A shy young man leaves his home in rural Turkey to learn a trade in 1920s Berlin. The city's crowded streets, thriving arts scene, passionate politics and seedy cabarets provide the backdrop for a chance meeting with a woman, which will haunt him for the rest of his life. Emotionally powerful, intensely atmospheric and touchingly profound, Madonna in a Fur Coat is an unforgettable novel about new beginnings and the unfathomable nature of the human soul.",
+      imgPath: 'https://i.dr.com.tr/cache/600x600-0/originals/0000000058245-1.jpg'
+    },
+    {
+      id: 9,
+      title: 'dava',
+      language: 'turkish',
+      author: 'franz_kafka',
+      desc:
+        "Dava, (Der Prozeß), bir sabah uyandığında kendisini sebebini anlamadığı bir suç nedeniyle dava edilmiş bulan Josef K. adlı kahramanın absürt durumunun anlatıldığı bir Franz Kafka romanıdır. Gerçekdışı niteliğiyle Kafka'nın şaşırtıcı yapıtları arasında çok önemli bir yeri olan Dava; tamamlanmamış bölümleriyle birlikte yazarın ölümünden iki yıl sonra, yakın arkadaşı Max Brod'un katkılarıyla, 1925'te yayımlanmıştır. ",
+      imgPath: 'https://www.iskultur.com.tr/dosyalar/2015/03/dava_10-255x420.jpg'
+    },
+    {
+      id: 10,
+      title: 'donusum',
+      language: 'turkish',
+      author: 'franz_kafka',
+      desc:
+        "Dönüşüm, Değişim veya Metamorfoz (Almanca özgün adı: Die Verwandlung), Franz Kafka'nın uzun öyküsü. İlk olarak 1915 yılında yayımlanmıştır.Kafka'nın en popüler eseri sayılabilir. Öykü, Gregor Samsa'nın bir sabah kendini dev bir böceğe dönüşmüş bulmasıyla başlar ve hayatındaki değişiklikleri anlatarak devam eder. ",
+      imgPath: 'https://i.dr.com.tr/cache/600x600-0/originals/0000000064022-1.jpg'
+    }
+  ];
+
+  function toCapitalize(text) {
+    return text
+      .toLowerCase()
+      .split(' ')
+      .map(s => s.charAt(0).toUpperCase() + s.substring(1))
+      .join(' ');
+  }
 
   // Replace with your own code
-  console.log(bookTitles);
+  function render(container) {
+    // for (const book of bookTitles) {
+    //   const li = document.createElement('li');
+    //   li.innerText = book;
+    //   container.appendChild(li);
+    // }
+
+    for (const title of books) {
```

Comment:
Be consistent in your naming. If you have an array of books, then, a single element is a `book`, not a `title`.